django_tables2_column_shifter/tests/test_base.py

```diff
@@ -45,13 +47,27 @@ class DjangoTables2ColumnShifterTest(TestCase):
             'template_name': 'django_tables2_column_shifter/bootstrap4.html',
             'table_clsss': ColumnShiftTableBootstrap4,
         },
+        {
+            'bootstrap_version': 'bootstrap4responsive',
+            'min_dt_version': (2, 5),
+            'max_dt_version': None,
+            'template_name': 'django_tables2_column_shifter/bootstrap4-responsive.html',
+            'table_clsss': ColumnShiftTableBootstrap4Responsive,
+        },
         {
             'bootstrap_version': 'bootstrap5',
-            'min_dt_version': (2, 0),
+            'min_dt_version': (2, 5),
             'max_dt_version': None,
             'template_name': 'django_tables2_column_shifter/bootstrap5.html',
             'table_clsss': ColumnShiftTableBootstrap5,
         },
+        {
+            'bootstrap_version': 'bootstrap5responsive',
+            'min_dt_version': (2, 5),
+            'max_dt_version': None,
+            'template_name': 'django_tables2_column_shifter/bootstrap5-responsive.html',
+            'table_clsss': ColumnShiftTableBootstrap5Responsive,
+        },
     ]
 
     dt_version = tuple(map(int, tables.__version__.split(".")[:2]))
```

Comment:
Can you remove `dt_version` from here and import it from  `django_tables2_column_shifter.tables ` at top of file. 
After yesterday changes we already have `dt_version` in `tables`
Moreover, now you need version containing 3 digits instead 2.